make.bat

```diff
@@ -39,14 +41,29 @@ if "%1" == "help" (
 	echo.  check
 	echo.  serve      to serve devguide on the localhost (8000)
 	goto end
-)
+:help0
```

Comment:
I'm not particularly familiar with bat files, and I'm not sure I understand this change.
ISTM that you just avoid printing the help if the arg is not "help", so the end result should be the same as before.
FWIW all the other commands follow the structure:
```
if "%1" == "something" (
    ...
    goto end
)
```
except "check" and "serve".
If possible, to maintain symmetry and improve readability, I would keep/adopt the same structure for all the different targets, and combine with an OR the two targets that result in an help ("help" and "").  This could be done in a separate PR, but unless there's a reason to break this structure, I would either revert or refactor this change.

(I made this comment on the first review, but apparently forgot to save/add it, so I'm adding it back now.)

Reply:
I'm not great with `.bat` files either, but as briefly stated in the related issue #267, the `.\make html` currently builds nothing because `goto end` is executed unconditionally. I attributed this to known issues with parenthetical `if` bodies and `goto`, anr the legacy-style work-around made it go away.

On closer inspection, the problem is the un-escaped `(8000)`. I can fix that.

There is no OR operator, but I can make it a bit clearer (closer to what `sphinx-quickstart` generated).

Reply:
If there's a valid reason to do it that way, then go ahead, but perhaps add a comment to document it :)

Reply:
Thanks to your challenge I have found the real problem and the right answer.